Makefile

```
@@ -282,6 +282,11 @@ else
 	npx react-native run-ios
 endif
 
+# Get all clojure files, including untracked, excluding removed
+define all_clojure_files
+$$(comm -23 <(sort <(git ls-files --cached --others --exclude-standard)) <(sort <(git ls-files --deleted)) | grep -e \.clj$$ -e \.cljs$$ -e \.cljc$$ -e \.edn)
```

We could make this into an actual function rather than just a definition of commands we want to run:
```suggestion
$(shell 
	comm -23 \
		<(sort <(git ls-files --cached --others --exclude-standard)) \
		<(sort <(git ls-files --deleted)) \
		| grep -e \.clj$$ -e \.cljs$$ -e \.cljc$$ -e \.edn
)
```
But then we need double quotes around the `$(call find_all_clojure_files)`.

---

Would be nice to accomplish this in one `git ls-files` command, but I wasn't able to. I hoped some combination of `--exclude=':S' would do it, but apparently not.

I haven't seen `comm` before - interesting!

---

I have the lingering feeling that this could all be done in one `ls-files` command, including the [`grep` part](https://stackoverflow.com/questions/36753573/how-do-i-exclude-files-from-git-ls-files).

But I am also getting rubik's cube vibes (seeing this as an interesting puzzle to solve) that no one else would be interested in or in maintaining. We might be more comfortable with this mix of `comm, sort, ls-files, grep` (which makes sense) over figuring out arguments to `ls-files`. However, if we have a lot of `ls-files` stuff to do, it might pay off.

If there's interest, I might pursue it, otherwise I'll move on to other things. @jakubgs

---

@jakubgs I'd keep it as it is for now, but if you feel strongly I can address in a separate PR (I really want to merge before the weekend ;) ) but I have address the rest of the points, thanks for the review!